tests/kokkos-based/overwriting_matrix_vector_product.cpp

```diff
@@ -51,7 +51,7 @@ void kokkos_blas_overwriting_gemv_impl(A_t A, x_t x, y_t y)
     // check A and y
     std::size_t count=0;
     for (std::size_t i=0; i<extent0; ++i){
-      EXPECT_FLOAT_EQ(y(i), y_gold(i));
```

Comment:
good eye :) I just tried with 1e-4 works, with 1e-6 does not. 
The reason i put 1e-2 was because i used that for another test. But yes you are right, I was hoping to get this merged and then revisit all tolerances.

Reply:
@fnrizzi No worries!